### PR TITLE
zh-translation:/zh/docs/reference/config/proxy_extensions/_index.md

### PR DESCRIPTION
**Pull request by dapengJacky**
[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by dapengJacky** [comment]
I signed it!

**Comment by dapengJacky** [comment]
@googlebot  I signed it!

File: content/zh/docs/reference/config/proxy_extensions/accesslogpolicy/index.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+source_repo: https://github.com/istio/proxy
+title: AccessLogPolicy配置
+description: AccessLogPolicy过滤器的配置。
+location: https://istio.io/docs/reference/config/proxy_extensions/accesslogpolicy.html
+layout: protoc-gen-docs
+generator: protoc-gen-docs
+weight: 20
+number_of_entries: 1
+---
+<p>Accesslog Policy plugin is a stateful http log sampler.
```
Comment (kebe7jun) [inline content/zh/docs/reference/config/proxy_extensions/accesslogpolicy/index.html:12]: 这些东西你都没翻译哦。
这是你期望的嘛？

**Review by SataQiu — COMMENTED** [review]
/ok-to-test

**Pull request closed without merging**